fe/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -1013,6 +1013,13 @@ public static void initColumns(Table tbl, List<ImportColumnDesc> columnExprs,
                 slotDescByName.put(realColName, slotDesc);
             }
         }
+        // The extension column of the materialized view is added to the expression evaluation of load
+        for (Column column : tbl.getFullSchema()) {
+            if (column.getDefineExpr() != null) {
```

Comment:
Are you sure that this is correct` column(a, tmp_c) set(b=f(c), c=b) `？

Reply:
> Are you sure that this is correct`column(a, tmp_c) set(b=f(c), c=b)`？

Fix done